app.js

```diff
@@ -1,17 +1,30 @@
 'use strict';
 
+var storeSalesDataEl = document.createElement('table');
+
+var timeRowEl = document.createElement('tr');
+storeSalesDataEl.appendChild(timeRowEl);
+
+var blankCell = document.createElement('td');
+timeRowEl.appendChild(blankCell);
+
+var sectionEl = document.getElementById('cookie_table_section');
+sectionEl.appendChild(storeSalesDataEl);
+
+var businessHours = ['06am','07am ', '08am ', '09am ', '10am ', '11am ', '12pm ', '01pm ', '02pm ', '03pm ', '04pm ', '05pm ','06pm ','07pm ', '08pm '];
+
 function CookieStore (name, minCustomers, maxCustomers, avgCookies, hourlyCookieSales) { // this is a constructor function taking in 4 parameters.
   this.name = name || 'Unknown'; // adding || (or) inputs a value if it doesn't meet true.
   this.minCustomers = minCustomers;  // the context of this is going to change. The keyword 'this' knows what it represents by the surrounding context
   this.maxCustomers = maxCustomers; // argument on the right is going into the parameter in the function.
   this.avgCookies = avgCookies;
   this.hourlyCookieSales = hourlyCookieSales || [];// assigning an empty array []
   this.totalCountForDay = 0;
-  this.businessHours = ['06am','07am ', '08am ', '09am ', '010am ', '011am ', '12pm ', '01pm ', '02pm ', '03pm ', '04pm ', '05pm ','06pm ','07pm ', '08pm ', 'Daily Location Total'];
+  this.businessHours = businessHours;
```

Comment:
I like this, now you can use businessHours instead of firstAndPike.businessHours later.